waspc/packages/wasp-config/src/appSpec.ts

```diff
@@ -0,0 +1,210 @@
+/** This module is a mirror implementation of AppSpec Decls in TypeScript.
```

Comment:
More specifically, they mirror the implementation of FromJSON for AppSpec Decls in Haskell, right?